### PR TITLE
Release 0.8.0

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ begin
     config.exclude_labels = %w{duplicate question invalid wontfix wont-fix modulesync skip-changelog}
```
Comment (bastelfreak) [inline Rakefile:5]: can you please add `github_actions` to the exclude list? We try to keep those automatic PRs for actions out of the changelog.

Comment (rwaffen) [reply Rakefile]: done

Comment (bastelfreak) [reply Rakefile]: you also need to regenerate the changelog

Comment (rwaffen) [reply Rakefile]: no changes

Comment (bastelfreak) [reply Rakefile]: run `rm -rf /tmp/github-changelog-*` and try again

Comment (rwaffen) [reply Rakefile]: after `github_action` seems to be a newer label. after manually adding it for old PRs its now acknowledged

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**